### PR TITLE
improved lisibility

### PR DESCRIPTION
**Pull request by kokounet**
using numpy overload on add, sub, multiply, divide, mod

### CODE REVIEW

**Comment by hernanat** [comment]
nice change! I appreciate you contributing to the project :)

File: dcolor.py [hunk 1]
```diff
@@ -24,19 +24,19 @@ def makeDomain(self):
 
     def makeColorModel(self, zz):
         """Create the HSV color model for the function domain that will be plotted"""
-        H = self.normalize(np.mod(np.angle(zz),2*np.pi)) #Hue determined by arg(z)
-        r = np.log2(np.add(1.0,np.abs(zz)))
-        S = np.divide(np.add(1.0,np.abs(np.sin(np.multiply(2.0*np.pi,r)))),2.0)
-        V = np.divide(np.add(1.0,np.abs(np.cos(np.multiply(2*np.pi,r)))),2.0)
+        H = self.normalize(np.angle(zz) % (2*np.pi)) #Hue determined by arg(z)
```
Comment (hernanat) [inline dcolor.py]: do we also want to add a space between the `*` here? Also, should it be `2.`? i.e. 

```python
(2. * np.pi)
```

File: dcolor.py [hunk 2]
```diff
@@ -24,19 +24,19 @@ def makeDomain(self):
...
+        r = np.log2(1. + np.abs(zz))
+        S = (1. + np.abs(np.sin(2.*np.pi * r))) / 2.
```
Comment (hernanat) [inline dcolor.py]: should we be consistent with spaces between operators? i.e. `2. * np.pi`

**Review by hernanat — CHANGES_REQUESTED** [review]
a few comments on consistency of spacing, otherwise looks good!

File: dcolor.py (hunk 2 above)
Comment (kokounet) [reply dcolor.py]: Yeah clearly! gonna fix this !

**Comment by hernanat** [comment]
nice work @chrisjanaqi !

**Pull request merged**